app/main.py

```diff
@@ -1,25 +1,67 @@
-from abc import ABC
+from abc import ABC, abstractmethod
+from typing import Any, Type, Tuple
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __get__(self, instance: Any, owner: Type) -> Tuple[int, int]:
+        if instance is None:
+            return self.min_amount, self.max_amount
+        return self.min_amount, self.max_amount
```

Comment:
It must return stored valu from `instance` instead of `self.min_amount, self.max_amount`